.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us squash bugs!
+labels: bug
+---
+<!--
+
+Please give us an overview of what went wrong, and what you expected to have
+happen instead.
+
+-->
+
+**Steps to reproduce**
+
+<!--
+
+Please provide step-by-step instructions as to how to reproduce the problem.
+
+-->
+
+**When can we consider this to be "fixed"?**
+
+<!--
+
+Please describe clear and practical acceptance criteria for this issue (i.e. a
+"definition of done"). Anyone in the team should be able to look at this issue
+and determine whether or not the bug's been fixed.
```

Comment:
```suggestion
and determine whether or not the bug has been fixed.
```

The grammar "soldier" in me...

Reply:
I usually purposefully put the contractions in nowadays to indicate less formality (being _Informal_ and all 😜).

Reply:
Fair point.